### DISCUSSION TITLE
Tagged Releases

### DISCUSSION THREAD

**Discussion by lukepistrol** [migueldeicaza/SwiftTerm#280] [General]
Hey, I wanted to ask whether or not it's possible to regularly create new release tags, so we (`CodeEdit`) can target a fixed version and still don't miss out on the great new functionality introduced in the last couple of years. I noticed that the latest tag (`1.0.7`) is actually almost 2 years old by now and for stability reasons we'd like to not target the `main` branch.

<img width="341" alt="Screenshot 2023-03-07 at 16 36 03" src="https://user-images.githubusercontent.com/9460130/223471472-d3b16bb2-2f34-4cf8-a433-0c0dd0f87c99.png">

**Comment by migueldeicaza** [comment]
Really good idea.

Let me add a 1.2.0 now

**Reply by lukepistrol** [reply]
Thanks a lot 😊
